components/builder-jobsrv/src/server/worker_manager.rs

```diff
@@ -252,10 +253,10 @@ impl WorkerMgr {
 
     fn process_heartbeat(&mut self) -> Result<bool> {
```

Comment:
I assume this will come later - if we abstract out the HB functionality, it would be nice to not have any knowledge of the HB implementation internals needed in the consuming modules.